pyproject.toml

```
@@ -66,11 +66,16 @@ fits = [
     "kerchunk>=0.2.8",
     "astropy",
 ]
+tif = [
```

Surely? (and everywhere else this occurs too)
```suggestion
tiff = [
```